app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Which means 🤔 The Post belongs-to user through the `'author_id'` column of the Post table. Now, we need to show that, the  `'author_id'` reference  User class modal.
Your code should look like this:

```ruby
  # app/models/post.rb
  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Reply:
Hello, @BenMukebo, 
Thanks for your review. Could you help to clarify one doubt? In my case, my reference column is user_id, not author_id, this different name has been already approved, so I have to do the same that you said to show a reference between Post and User.
Thanks for your help.

Reply:
Hello @VaneCode !!

Reply:
Hello @BenMukebo,
Thanks for your explanation, but I got your idea from the beginning, and right now I have more doubts:
-It called user_id instead of author_id is wrong, why they approved the previous project?
-There is a technical reason to use author_id, because I think is more clear to use user_id as a name for the foreign key.
-If a run the commands that you suggest, there is not going to affect the data and the structure I already have. Because I think those commands are to add a new foreign key, and I already have them but whit a different name.
Please help with these doubts to strengthen my knowledge about Ruby, 
Thanks.

Reply:
Hi @VaneCode 
`Author` model acts as an _alias_, or a through relationship for `User` model, and hence, following the ERD, you should use `author_id` accordingly for FK or referencing `User` model.